WalletLibrary/WalletLibrary/VerifiedIdClientBuilder.swift

```diff
@@ -26,7 +26,9 @@ public class VerifiedIdClientBuilder {
         let _ = VCServices.VerifiableCredentialSDK.initialize()
         
         let configuration = LibraryConfiguration(logger: logger,
-                                                 mapper: Mapper())
+                                                 mapper: Mapper(),
+                                                 verifiedIdDecoder: VerifiedIdDecoder(),
```

Comment:
Is it instantiated both here and Configuration? Is the one in configuration like default value?

Reply:
Yea I just put it in default values for testing but theoretically, in the future we can inject the logger or something into the decoder and encoder here.